fairscale/nn/data_parallel/__init__.py

```diff
@@ -8,4 +8,5 @@
 from .fully_sharded_data_parallel import FullyShardedDataParallel, OffloadConfig, TrainingState, auto_wrap_bn
 from .sharded_ddp import ShardedDataParallel
 
+# FIXME: why the empty __all__ list with explicit imports?
```

Comment:
oh, yeah, I remember now. I think we debated a bit on whether we want to support:

```
from foo import *
```

I think the `__all__`, only controls that case. See this bullet point in CONTRIBUTING.md to see the rational behind it:
![image](https://user-images.githubusercontent.com/24926999/171028315-31732a19-a8ed-4be7-ae76-fa38548fc4b4.png)


With empty __all__, import * won't accidentally import symbols that covers other symbols with the same name. Also, it lower the overhead for devs to maintain 2 list in a consistent way.